packages/react-components/react-table/src/components/TableCellLayout/useTableCellLayoutContextValues.ts

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import type { TableCellLayoutState, TableCellLayoutContextValues } from './TableCellLayout.types';
+
+const tableAvatarSizeMap = {
+  medium: undefined,
+  small: 24,
+  smaller: 20,
+} as const;
+
+export function useTableCellLayoutContextValues_unstable(state: TableCellLayoutState): TableCellLayoutContextValues {
+  const { size: tableSize } = state;
+
+  const avatar = React.useMemo(
+    () => ({
+      size: tableAvatarSizeMap[tableSize],
```

Comment:
IMO, a value should be stored in `state` so it could be overridden with composition patterns.

Reply:
you're right, I added `avatarSize` to state